HW03/Kate Fedotova/student.rb

```diff
@@ -0,0 +1,25 @@
+class Student
+  attr_accessor :name, :surname, :homeworks, :mentor
+
+  def initialize(name, surname)
+    @name = name
+    @surname = surname
+    @homeworks = []
+  end
+
+  def submit_homework(new_hw)
+    homework = Homework.new(new_hw)
+    @homeworks << homework
+    notification_message = "#{name} #{surname} submit homework: #{new_hw}"
+    send_notification(notification_message)
+    puts homeworks.to_s
+  end
```

Comment:
Maybe you should pass the Homework object as a parameter to this method? Imagine that the class homework will change, you have extra dependencies.
```suggestion
  def submit_homework(new_hw, homework = Homework)
    @homeworks << homework
    notification_message = "#{name} #{surname} submit homework: #{new_hw}"
    send_notification(notification_message)
    puts homeworks.to_s
  end
```
P.s This is pseudo-code; it may be incorrect))))